packages/plugin-user/src/user.js

```diff
@@ -274,15 +274,20 @@ const User = SparkPlugin.extend({
         })
         .catch((err) => Promise.reject(new Error(`failed to set authorization`, err)))
       )
-      .then(() => this.request({
-        service: `atlas`,
-        resource: `users/activations`,
-        method: `POST`,
-        headers,
-        body: options,
-        shouldRefreshAccessToken,
-        requiresClientCredentials
-      }))
+      .then(() => {
+        if (options.spoofUserAgent) {
```

Comment:
nevermind, I see you are just checking the headers in case someone sets it somewhere. headers is initialized just above so I don't think that is necessary.